.github/workflows/github-actions-test-installer.yml

```diff
@@ -2,14 +2,16 @@ name: Test DependencyInstaller script
 on:
   schedule:
   - cron: "0 8 * * MON"
+  push:
+  pull_request:
```

Comment:
The following might be a good improvement, it should be triggered on etc/DependencyInstaller and docker/* at least. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files